API-Samples/Java/gradlew

```diff
@@ -0,0 +1,240 @@
+#!/bin/sh
```

Comment:
You need to git exclude the gradle-wrapper.jar above.

Reply:
https://github.com/github/gitignore/blob/main/Gradle.gitignore

# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)